tests/PetrolCar.ut.cpp

```diff
@@ -0,0 +1,29 @@
+#include <gtest/gtest.h>
+#include "PetrolCar.hpp"
+
+constexpr int enginePower = 100;
+constexpr float engineCapacity = 8.0;
+constexpr int engineGears = 6;
+
+constexpr int betterEnginePower = 100;
```

Comment:
it's the same as enginePower. Comparing them will result in true. Something that we would like to avoid.